public/app/core/components/Select/DashboardPicker.tsx

```diff
@@ -46,25 +47,21 @@ export class DashboardPicker extends PureComponent<Props, State> {
   };
 
   render() {
-    const { className, onSelected, currentDashboardId } = this.props;
+    const { size, onSelected, currentDashboardId } = this.props;
     const { isLoading } = this.state;
 
     return (
-      <div className="gf-form-inline">
-        <div className="gf-form">
-          <AsyncSelect
-            className={className}
-            isLoading={isLoading}
-            isClearable={true}
-            defaultOptions={true}
-            loadOptions={this.debouncedSearch}
-            onChange={onSelected}
-            placeholder="Select dashboard"
-            noOptionsMessage={() => 'No dashboards found'}
-            value={currentDashboardId}
-          />
-        </div>
-      </div>
+      <Forms.AsyncSelect
+        size={size}
```

Comment:
```suggestion
        size={size || 'md'}
```
Probably needs a default fallback, missed this the first time
![Screenshot from 2020-03-03 14-25-10](https://user-images.githubusercontent.com/1438972/75779962-e940cf80-5d5a-11ea-91f8-4f32a7cbad2d.png)

Reply:
@tskarhed  Yeah, good point, altho I think these fallbacks are better handled via `defaultProps`/default params, so they are all in one place :) 

I already have a PR that migrates the rest of the ReportPage to the new form elements, just needed this one to get merged first.

Reply:
Good point, that's awesome too! Can you add the default prop in this PR?

Reply:
Done :)